packages/components/avatar/src/Avatar.tsx

```diff
@@ -4,35 +4,35 @@ import { type CommonProps } from '@contentful/f36-core';
 import { Image, type ImageProps } from '@contentful/f36-image';
 import { getAvatarStyles } from './Avatar.styles';
 
-export enum Size {
-  Tiny = '20px',
-  Small = '24px',
-  Medium = '32px',
-  Large = '48px',
-}
+export type Size = 'tiny' | 'small' | 'medium' | 'large';
 
-export enum Variant {
-  App = 'app',
-  User = 'user',
-}
+export type Variant = 'app' | 'user';
 
 export interface AvatarProps extends CommonProps {
   alt?: ImageProps['alt'];
   isLoading?: boolean;
   size?: Size;
   src?: ImageProps['src'];
   variant?: Variant;
+  showColorBorders?: boolean;
```

Comment:
```suggestion
  showColorBorder?: boolean;
```

I think singular `showColorBorder` fits better here, it's also what you've used yourself in a few places ;D